Assets/Scripts/Player/PlayerController.cs

```diff
@@ -10,51 +10,59 @@
 using UnityEngine.InputSystem;
 
 /// <summary>
-/// Handles player input
+/// Handles player input and movement
 /// </summary>
 public class PlayerController : MonoBehaviour
 {
-    private StateMachine playerStateMachine;
+    private PlayerStateMachine playerStateMachine;
     public Rigidbody2D rb;
     // Reference to player stats script
     private PlayerStats player;
 
     private float moveSpeed;
+    private float attackSpeed;
+    private float strength;
+    private float healthPoints;
     // Vector2 of the normalized vectors of movement for the player
```

Comment:
Are we thinking of including the other player attributes at a later time? ie. Defence, Incoming Damage and the enum Player Status?

Reply:
That was my thinking, but I can add them if needed

Reply:
I added them